presto-verifier/src/test/java/com/facebook/presto/verifier/framework/TestDataVerification.java

```diff
@@ -347,6 +363,17 @@ private void assertEvent(
             Optional<DeterminismAnalysis> expectedDeterminismAnalysis,
             Optional<String> expectedErrorCode,
             Optional<String> expectedErrorMessageRegex)
+    {
+        assertEvent(event, expectedStatus, expectedDeterminismAnalysis, expectedErrorCode, expectedErrorMessageRegex, true);
+    }
+
+    private void assertEvent(
+            VerifierQueryEvent event,
+            EventStatus expectedStatus,
+            Optional<DeterminismAnalysis> expectedDeterminismAnalysis,
+            Optional<String> expectedErrorCode,
+            Optional<String> expectedErrorMessageRegex,
+            boolean controlWasRun)
```

Comment:
can we rename to `isControlQueryExecuted` so that we follow the trend of using `is` for boolean values.

Reply:
Renamed.